src/app/ApplicationTemplate.Shared.Views/Framework/ExtendedSplashscreenController.cs

```diff
@@ -1,22 +1,23 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using ApplicationTemplate.Views;
 using Microsoft.UI.Dispatching;
 
 namespace ApplicationTemplate;
 
 public class ExtendedSplashscreenController : IExtendedSplashscreenController
 {
-	private readonly DispatcherQueue _coreDispatcher;
+	private DispatcherQueue _dispatcher;
 
-	public ExtendedSplashscreenController(DispatcherQueue coreDispatcher)
+	public ExtendedSplashscreenController(DispatcherQueue dispatcherQueue)
 	{
-		_coreDispatcher = coreDispatcher;
+		_dispatcher = dispatcherQueue;
 	}
 
-	public void Dismiss()
+	public async void Dismiss()
```

Comment:
never use `async void`

Reply:
This seems unrelated to the packages clean-up

Reply:
Those changes were pushed by mistake. I reverted and kept the changes related to this PR